addons/missileGuidanceCompat/patchCWRBAF/config.cpp

```diff
@@ -27,13 +27,25 @@ class CfgAmmo {
             seekerMaxRange = 6000;
         };
     };
+    class GVAR(blowpipe_a): CUP_M_Stinger_AA {
+        class ace_missileguidance: ACEGVAR(missileguidance,type_Blowpipe) {
+            enabled = 1;
+        };
+    };
 };
 
 class CfgMagazines {
     class CA_LauncherMagazine;
     class cwr3_javelin_m: CA_LauncherMagazine {
         ammo = QGVAR(M_javelin_AA);
```

Comment:
```suggestion
        ammo = QGVAR(M_javelin_AA);
        ACEGVAR(missile_manpad,enabled) = 0;
```
Will need this for them to work with the Stinger PR

Reply:
it's now an explicit config check, also on weapon not mag
https://github.com/acemod/ACE3/pull/10830/files#diff-a107455f1cee38185ba9f752dce8335f1df99db55070d392d1d2e85445474560R32

Reply:
Sounds good, which PR do I need to review this one?

Reply:
I think just https://github.com/acemod/ACE3/pull/10887 (I guess 10893 would be neat but not at all needed)